docs/content/guide/migration.ngdoc

```diff
@@ -1226,6 +1226,16 @@ or simply use:
   As communicated before, IE8 is no longer supported.
 
 
+### Filters (`orderBy`)
+
+- due to [a097aa95](https://github.com/angular/angular.js/commit/a097aa95b7c78beab6d1b7d521c25f7d9d7843d9),
+  the default orderBy comparator sorts null values (strings, objects, etc.) as if they were the string `'null'`:
```

Comment:
Hm...actually things are a little more complicated: https://github.com/angular/angular.js/issues/15293#issuecomment-255190763

For this breaking change note, you need to mention the change introduced in a097aa95.
For the docs, you need to mention the current situation, as introduced in 48e1f56.